src/components/SearchForm.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react";
+
+const SearchForm = ({ callback, value }) => {
+  const [searchValue, setSearchValue] = useState(value);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    callback(searchValue);
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <input
+        type="text"
+        value={searchValue}
+        onChange={(e) => setSearchValue(e.target.value)}
+        onKeyDown={(e) => {
+          if (e.key === "Enter") handleSubmit(e);
+        }}
+      />
+      <button type="submit" onClick={handleSubmit}>
```

Comment:
```suggestion
      <button type="submit">
```
this onClick is redundant as we passed already this function In onSubmit